### DISCUSSION TITLE
Need help!!!!.

### DISCUSSION THREAD

**Discussion by AaronRiboy** [serenity-is/Serenity#6675] [Help]
Hello guys, im new to this framework and this is my 1st typescript project as well. I have  2 yrs of .net core experience but having difficulty to enhance some function in this framework. Can anyone  clear my doubt through video call ?Im willing to pay .Currently my office project on my hand now . Please .Thank you

**Comment by edwardch** [comment]
Please can you try to describe your problem as it will make it possible for someone to help. 

Right now there are no details so nobody can help, and not knowing the extent of the problem would also make it less likely for someone to agree to a video call.

On the front page of the GitHub repo you'll see a "Contact us" link where you can ask the Serenity team for paid support. 

BTW I'm just a community member and I try to help others where I can.
